ENIGMAsystem/SHELL/Platforms/General/PFmain.h

```diff
@@ -20,6 +20,17 @@
 #define ENIGMA_PLATFORM_MAIN
 
 #include <string>
+#include <queue>
+#include <map>
+
+#include "Universal_System/var4.h"
+
+namespace enigma {
+  struct extension_steamworks
+  {
+    virtual variant myevent_asyncsteam() { return 0; }
```

Comment:
@RobertBColton 

I shouldn't do that right? as this function is auto-generated, what do u think?

Reply:
We talked about this, yes you do actually do this. The struct is named in the `.ey` file and that's why calling it will cause the right event to be fired.

To quote Josh:
>are you asking why adding it to some meaningless struct causes it to be callable for all instances?
because you mention that struct in the .ey file I add that struct to the inheritance list of object_locals, giving it a vtable entry for those routines

Reply:
Done